lib/trento/domain/sap_system/database.ex

```diff
@@ -14,6 +14,7 @@ defmodule Trento.Domain.SapSystem.Database do
   deftype do
     field :sid, :string
     embeds_many :instances, Instance
+    field :deregistered_at, :utc_datetime_usec, default: nil
```

Comment:
I know that this is coming from other parts of the code, but the `default` option strucked me out.
I would say all fields are `nil` by default. Just for curiosity I wonder if having this default again does something hehe

Reply:
It's redundant, I removed it :D